icons/arrow.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="48" height="48" viewBox="0 0 48 48" fill="none" xmlns="http://www.w3.org/2000/svg">
+<g id="Frame 2609178">
+<rect width="48" height="48" fill="#B3976B"/>
+<path id="Vector 65" d="M14 23C13.4477 23 13 23.4477 13 24C13 24.5523 13.4477 25 14 25V23ZM32.7071 24.7071C33.0976 24.3166 33.0976 23.6834 32.7071 23.2929L26.3431 16.9289C25.9526 16.5384 25.3195 16.5384 24.9289 16.9289C24.5384 17.3195 24.5384 17.9526 24.9289 18.3431L30.5858 24L24.9289 29.6569C24.5384 30.0474 24.5384 30.6805 24.9289 31.0711C25.3195 31.4616 25.9526 31.4616 26.3431 31.0711L32.7071 24.7071ZM14 25L32 25V23L14 23V25Z" fill="#1D1D1D"/>
```

Comment:
Can you export the icon without the background? We need the button to be styled with different states, as you can find here: https://www.figma.com/file/URJhT8pkntqc6h6v5wSLqx/Mack-Core-Design-Kit?type=design&node-id=49-4333&mode=dev

Reply:
Updated